Tests/Runtime/Utilities/LogMessageHandlerTest.cs

```diff
@@ -152,6 +152,36 @@ public async Task HandleLog_ContainsIgnoreMessage_notReported()
             Assert.That(spyReporter.Arguments, Is.Empty);
         }
 
+        [Test]
+        public async Task HandleLog_MatchIgnoreMessagePattern_notReported()
+        {
+            var settings = CreateEmptyAutopilotSettings();
+            var spyReporter = ScriptableObject.CreateInstance<SpyReporter>();
+            var sut = new LogMessageHandler(settings, spyReporter);
+
+            settings.ignoreMessages = new[] { "ignore.+ignore" };
+            settings.handleException = true;
+            sut.HandleLog("ignore_xxx_ignore", string.Empty, LogType.Exception);
+            await UniTask.NextFrame();
+
+            Assert.That(spyReporter.Arguments, Is.Empty);
+        }
+
+        [Test]
+        public async Task HandleLog_NotMatchIgnoreMessagePattern_Reported()
+        {
+            var settings = CreateEmptyAutopilotSettings();
+            var spyReporter = ScriptableObject.CreateInstance<SpyReporter>();
+            var sut = new LogMessageHandler(settings, spyReporter);
+
+            settings.ignoreMessages = new[] { "ignore.+ignore" };
+            settings.handleException = true;
+            sut.HandleLog("ignore", string.Empty, LogType.Exception);
+            await UniTask.NextFrame();
+
+            Assert.That(spyReporter.Arguments, Is.Not.Empty);
+        }
+
```

Comment:
![must](https://img.shields.io/badge/review-must-red.svg)

There isn't a test case when invalid regex are entered such as "[a".
Add the test case or tell me how it behaves in the case, please.

Reply:
Fixed in 4ee60f362c20ab7e7740b93ede8a1ddbe9d6daef.

1. Add catch exception in `LogMessageHandler`, Do not throw an exception, only output to the logger.
2. Add test case